EIPS/eip-1123.md

```diff
@@ -3,13 +3,15 @@ eip: 1123
 title: Revised Ethereum Smart Contract Packaging Standard
 author: g. nicholas d’andrea (@gnidan), Piper Merriam (@pipermerriam), Nick Gheorghita (@njgheorghita), Danny Ryan (@djrtwo)
 discussions-to: https://github.com/ethereum/EIPs/issues/1123
-status: Draft
+status: Abandoned
 type: Standards Track
 category: ERC
 created: 2018-06-01
 replaces: 190
+superseded-by: 2678
 ---
 
+This ERC has been abandoned in favor of the EthPM V3 smart contract packaging standard defined in [ERC 2678](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2678.md)
```

Comment:
I know this can be annoying, but the current standard is:
```suggestion
This ERC has been abandoned in favor of the EthPM V3 smart contract packaging standard defined in [ERC-2678](https://eips.ethereum.org/EIPS/eip-2678)
```

Reply:
Can you apply this change? It was rolled back I think.